Dockerfile

```diff
@@ -47,4 +47,4 @@ RUN ./bootstrap.sh
 # CMD export PATH="$PATH:node_modules/.bin"
 # CMD export NVM_DIR="$HOME/.nvm"
 # CMD $NVM_DIR/nvm.sh
-CMD Toolchains/swift-5.7-RELEASE.xctoolchain/usr/bin/swift run -c release --static-swift-stdlib --build-path .build/swift-5.7-RELEASE
+CMD Toolchains/swift-5.7-RELEASE.xctoolchain/usr/bin/swift run -c release --scratch-path .build/swift-5.7-RELEASE PlaygroundServer serve --hostname 0.0.0.0
```

Comment:
* Build fails when stdlib is linked statically (on aarch64), I am not sure about the consequences though
* Default hostname is 127.0.0.1, which is not accessible outside of the container